README.md

```diff
@@ -215,6 +223,245 @@ objectives:
+| true AND false                  | false         |
+| false AND false                 | false         |
+| true AND indeterminate          | indeterminate |
+| indeterminate AND indeterminate | indeterminate |
+| true OR true                    | true          |
+| true OR false                   | true          |
+| false OR false                  | false         |
+| true OR indeterminate           | true          |
+| indeterminate OR indeterminate  | indeterminate |
+| false OR indeterminate          | indeterminate |
+
+In the table below lists when alert policy will be triggering depending on
+the resolved alert condition value:
+
+| Alert condition | alertIf | Alert triggering |
+|---|---|---|
+| true | true | Yes |
+| true | false | No |
+| true | indeterminate | No |
+| false | true | Yes |
+| false | false | No |
+| false | indeterminate | No |
+| indeterminate | true | No |
+| indeterminate | false | No |
+| indeterminate | indeterminate | Yes |
+
+---
+
+#### Alert Condition
+
+An Alert Condition allows you to define in which conditions a alert of SLO
+needs to be triggered.
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertCondition
+metadata:
+  name: string
+  displayName: string # optional
+spec:
+  description: # optional
+  severity: string # required (ticket or page)
+  condition: # optional
+    kind: string
+    threshold: number
+    lookbackWindow: number
+    alertAfter: number
+```
+
+#### Notes (Alert Condition)
+
+- **severity** *enum(ticket, page)*, required field. The severity level of the alert
+- **condition**, required field. Defines the conditions of the alert
+  - **kind** *enum(burnrate, custom)* the kind of alerting condition thats checked, defaults to `burnrate`
+
+If the kind is `burnrate` the following fields are required:
+
+- **threshold** *number*, required field, the threshold that you want alert on
+- **lookbackWindow** *number*, required field, the time-frame for which to calculate the threshold
+- **alertAfter** *number*: required field, the duration the condition needs to be valid, defaults `0m`
+
+If the kind is `custom` the following fields are required:
+
+- **threshold** *number*, required field, the threshold that you want alert on
+- **comparison** *enum(lt, lte, gt, gte, eq)*, optional field, defines
+  how the threshold should be compared to meet the threshold, defaults to `gt`
+
+If the alert condition can not be resolved to a `true` or `false` value, for example
+due to missing data points, then the `indeterminate` needs to be returned.
+
+---
+
+An example of a alert condition is the following:
+
+```yaml
+apiVersion: openslo/v1alpha
+kind: AlertCondition
+metadata:
+  name: cpu-usage-breach
+  displayName: CPU usage breach
+spec:
+  description: If the CPU usage is too high for given period then it should alert
+  severity: page
+  condition:
+    kind: burnrate
+    threshold: 0.9
+    lookbackWindow: 1h
+    alertAfter: 5m
+```
+
+---
+
+#### Alert Notification Target
```

Comment:
💡 Should we leave this as an implementation detail and out of scope of the Spec until we have a better understanding of the alerting flow? And we may be set an `notificationTarget` (or similar) on the alerting policy that can be a string with any value (email, PagerDuty, Cloudwatch, myCustomStuff) ?

Then when someones implement alerting targets they can pass metadata via annotations, etc..?

Reply:
Yeah, that makes sense. That simplifies this for now. What do you think @ian-bartholomew @nobl9-mikec ?

Reply:
That makes sense